src/raster/r.survey/r.survey.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```

Comment:
https://trac.osgeo.org/grass/wiki/Submitting/Docs#HTMLPages
--> _Important: The HTML file contains no header nor footer._

Reply:
Sorry Markus
My Fault.
We uploaded an old version due to a misleading
Now I put back the old one and corrected a couple of errors/warning